src/libs/actions/Session/index.ts

```diff
@@ -763,8 +775,11 @@ function invalidateCredentials() {
 }
 
 function invalidateAuthToken() {
-    NetworkStore.setAuthToken('pizza');
-    Onyx.merge(ONYXKEYS.SESSION, {authToken: 'pizza'});
+    // expires the session after 50s
+    setTimeout(() => {
+        NetworkStore.setAuthToken('pizza');
+        Onyx.merge(ONYXKEYS.SESSION, {authToken: 'pizza', encryptedAuthToken: 'pizza', creationDate: new Date().getTime() - CONST.SESSION_EXPIRATION_TIME_MS});
+    }, 50000);
```

Comment:
```suggestion
NetworkStore.setAuthToken('pizza');
Onyx.merge(ONYXKEYS.SESSION, {authToken: 'pizza', encryptedAuthToken: 'pizza', creationDate: 1});
```

Reply:
you have suggested that we use the testool "invalidate session" to do the test of expiring the session on native platforms because we could not use the devtools console in those platforms to do so. The test in the devtools console is to set the session creation date to 2hours before. If we do otherwise it's inconsistent with our web test.

Reply:
Well, the creationDate only need to be an invalid (expired) value, what's the matter?

Reply:
no, it need to be a realistic test. In the real life situation that this PR is fixing the session expires when its creation time iis more than 2h ago. Please let's do things the way it should reflect the real usage of app.

Reply:
QA can access that test tool menu on "real life" app. No need to worry about it.

Reply:
i'm speaking of what expiring a session means in real life usage of the app. It means its creation time is 2h ago. Any other value of that creation date is not a test of our PR.

Reply:
This is test tool, who cares about real life usage? A Testtool should do what it should. We need invalid session, that's all.

Reply:
no @hungvu193 we need to expire the session. We can invalidate just by only sending "pizza" as token but expiring the session has to do with the value of its creation date.

Reply:
Ok what ever. Let's remove timeout.